### ISSUE TITLE
[BUG] Tooltip shows briefly when using delayShow and moving to directly adjacent element

### ISSUE THREAD

**Issue by viveleroi** [ReactTooltip/react-tooltip#1162]
It seems that when using `delayShow`  and moving from one tooltip element to another that's immediately next to it, the "new" tooltip for the element you just moved to flashes for a second, hides, and then shows again after the time delay. Removing your mouse entirely and trying again works fine. 

So it seems to be related to moving from one tooltip element to another.

```tsx
    <Tooltip
      className={styles.tooltip}
      delayShow={1000}
      id='default'
      noArrow
      place='top-start'
    />
    
<span
  data-tooltip-content='This is a tooltip'
  data-tooltip-id='default'
  data-tooltip-variant='info'>test</span>
<span
  data-tooltip-content='This is a tooltip'
  data-tooltip-id='default'
  data-tooltip-variant='info'>test</span>
```

**Comment by Carnageous** [comment]
I was able to [reproduce that issue](https://stackblitz-starters-w5xzvh.stackblitz.io) while trying to debug another issue I am having with delay-show / delay-hide:

When you define a different show / hide delay on two elements and alternate hovering between them, the "show" delay from the previous element is applied, not the one from the currently hovering one.
https://stackblitz.com/edit/stackblitz-starters-dfebpx?file=src%2FApp.tsx

I`ll try to find the cause for this, but I would like to know if this is expected. Its not documented as far as I can see.

**Comment by gabrieljablonski** [comment]
`react-tooltip@5.26.1-beta.1168.0` (from #1168) seems to fix both these issues.

Feel free to test it out until we are ready to release an official version.

**Comment by gabrieljablonski** [comment]
Fixed on official release `react-tooltip@5.26.1`
